customer-terminate-service/app/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -116,4 +116,12 @@
     </addColumn>
   </changeSet>
 
+  <changeSet id="6" author="mh">
```

Comment:
How will this impact any already existing terminations as it cannot be null? Add a default value like change@me.se or something?